pkg/apis/numaflow/v1alpha1/udf.go

```diff
@@ -122,13 +122,21 @@ type GroupBy struct {
 type Window struct {
 	// +optional
 	Fixed *FixedWindow `json:"fixed" protobuf:"bytes,1,opt,name=fixed"`
+	// +optional
+	Sliding *SlidingWindow `json:"sliding" protobuf:"bytes,1,opt,name=sliding"`
 }
 
 // FixedWindow describes a fixed window
 type FixedWindow struct {
 	Length *metav1.Duration `json:"length,omitempty" protobuf:"bytes,1,opt,name=length"`
 }
 
+// SlidingWindow describes a sliding window
+type SlidingWindow struct {
+	Length *metav1.Duration `json:"length,omitempty" protobuf:"bytes,1,opt,name=length"`
+	Slide  *metav1.Duration `json:"slide,omitempty" protobuf:"bytes,1,opt,name=slide"`
```

Comment:
should be `2`

Reply:
corrected it later. in the latest commit.